src/screens/Asset/Asset.js

```diff
@@ -104,6 +107,7 @@ type Props = {
   history: Array<*>,
   logScreenView: (contentName: string, contentType: string, contentId: string) => void,
   availableStake: number,
+  contactsSmartAddresses: ContactSmartAddresses[],
```

Comment:
Type should be `ContactSmartAddresses` or `ContactSmartAddress`

Reply:
After discussions renamed to `ContactSmartAddressData`.